docs/architecture.md

```diff
@@ -1,32 +1,7 @@
-> The official Splunk documentation for this page is [Use Splunk Distribution of OpenTelemetry Collector](https://docs.splunk.com/Observability/gdi/opentelemetry/resources.html). For instructions on how to contribute to the docs, see [CONTRIBUTING.md](../CONTRIBUTING#documentation.md).
+The official Splunk documentation for this page is [Overview of the Splunk Distribution of OpenTelemetry Collector](https://docs.splunk.com/Observability/gdi/opentelemetry/resources.html). 
 
-# Architecture
+See architecture diagrams and deployment modes in [Collector deployment modes](https://docs.splunk.com/Observability/gdi/opentelemetry/deployment-modes.html).
 
-This distribution can be deployed in the below ways.
+For instructions on how to contribute to the docs, see [CONTRIBUTING.md](../CONTRIBUTING#documentation.md).
```

Comment:
```suggestion
For instructions on how to contribute to the docs, see [CONTRIBUTING.md](../CONTRIBUTING.md#documentation).
```

Reply:
Looks like this is the case elsewhere, but I wonder if we should refer to the public docs repo instead of this project's contributing guide.

Reply:
Thanks for catching this!